presto-parquet/src/main/java/io/prestosql/parquet/reader/MetadataReader.java

```diff
@@ -203,11 +205,12 @@ private static void readTypeSchema(Types.GroupBuilder<?> builder, Iterator<Schem
         }
     }
 
-    public static org.apache.parquet.column.statistics.Statistics<?> readStats(Optional<String> fileCreatedBy, Statistics statistics, PrimitiveType type)
+    public static org.apache.parquet.column.statistics.Statistics<?> readStats(Optional<String> fileCreatedBy, @Nullable Statistics statistics, PrimitiveType type)
```

Comment:
Maybe use `Optional<Statistics>` and have the caller do `Optional.ofNullable(...)`.

Reply:
good idea